### PR TITLE
update coreth flags

### PR DESCRIPTION
**Pull request by ceyonur**
# Docs PR Template

## Why this should be merged

Updates coreth flags with new ones and deprecated ones

## How this works

Updates chain-config-flags doc.

## How these changes were tested 

Tested on deployed test website.

## To prevent any errors while building

- [x] run `vale /docs/file/path` on all changed `.md` files to ensure all grammar rules pass
- [] run `markdownlint /docs/file/path` on all changed `.md` files to ensure all linting rules pass
- [] complete the above two checks and all additional rules outlined in `style-checker-notes.md` to
  ensure all checks pass

## SHOULD NOT BE MERGED UNTIL AvalancheGo v1.10.16

### CODE REVIEW

**Review by darioush — APPROVED** [review]

**Comment by meaghanfitzgerald** [comment]
/gt docs/nodes/configure/chain-config-flags.md i18n/es/docusaurus-plugin-content-docs/current/nodes/configure/chain-config-flags.md spanish

File: i18n/es/docusaurus-plugin-content-docs/current/nodes/configure/chain-config-flags.md [hunk 1]
```diff
@@ -154,16 +159,18 @@ Use el campo `eth-apis` para especificar el conjunto exacto de servicios a habil
...
+:::nota
 
 Los nombres utilizados en esta bandera de configuración se han actualizado en Coreth `v0.8.14`.
 Los nombres anteriores que contienen `public-` y `private-` están obsoletos. Si bien
 la versión actual continúa aceptando valores obsoletos, es posible que no se admitan
 en futuras actualizaciones y se recomienda actualizar a los nuevos valores.
 
-La asignación de valores obsoletos y su equivalente actualizado es la siguiente:
+La siguiente es la correspondencia de los valores obsoletos y su equivalente actualizado:
+
```
Comment (meaghanfitzgerald) [inline i18n/es/docusaurus-plugin-content-docs/current/nodes/configure/chain-config-flags.md:170]:
```suggestion
La asignación de valores obsoletos y su equivalente actualizado es la siguiente:
```

File: i18n/es/docusaurus-plugin-content-docs/current/nodes/configure/chain-config-flags.md [hunk 2]
```diff
@@ -358,8 +364,8 @@ Agrega las siguientes llamadas RPC al espacio de nombres `txpool_*`. El valor pr
 #### `internal-debug`
 
 Los nombres de API `internal-private-debug` e `internal-public-debug` están
-obsoletos a partir de la versión 1.7.15, y las API que antes estaban bajo estos nombres se han
-migrado a `internal-debug`.
+obsoletos a partir de la versión 1.7.15, y las API que antes se encontraban
+bajo estos nombres se han migrado a `internal-debug`.
```
Comment (meaghanfitzgerald) [inline i18n/es/docusaurus-plugin-content-docs/current/nodes/configure/chain-config-flags.md:368]:
```suggestion
obsoletos a partir de la versión 1.7.15, y las API que antes estaban bajo estos nombres se han
migrado a `internal-debug`.
```

File: i18n/es/docusaurus-plugin-content-docs/current/nodes/configure/chain-config-flags.md [hunk 3]
```diff
@@ -392,7 +398,7 @@ Agrega las siguientes llamadas RPC al espacio de nombres `debug_*`. El valor pre
 #### `internal-account`
 
 El nombre de API `internal-public-account` está obsoleto a partir de la versión 1.7.15, y las API
-que antes estaban bajo este nombre se han migrado a `internal-account`.
+que antes se encontraban bajo este nombre se han migrado a `internal-account`.
```
Comment (meaghanfitzgerald) [inline i18n/es/docusaurus-plugin-content-docs/current/nodes/configure/chain-config-flags.md:401]:
```suggestion
que antes estaban bajo este nombre se han migrado a `internal-account`.
```

File: i18n/es/docusaurus-plugin-content-docs/current/nodes/configure/chain-config-flags.md [hunk 4]
```diff
@@ -401,7 +407,7 @@ Agrega las siguientes llamadas RPC al espacio de nombres `eth_*`. El valor prede
 #### `internal-personal`
 
 El nombre de API `internal-private-personal` está obsoleto a partir de la versión 1.7.15, y las
-API que antes estaban bajo este nombre se han migrado a `internal-personal`.
+API que antes se encontraban bajo este nombre se han migrado a `internal-personal`.
```
Comment (meaghanfitzgerald) [inline i18n/es/docusaurus-plugin-content-docs/current/nodes/configure/chain-config-flags.md:410]:
```suggestion
API que antes estaban bajo este nombre se han migrado a `internal-personal`.
```

File: i18n/es/docusaurus-plugin-content-docs/current/nodes/configure/chain-config-flags.md [hunk 5]
```diff
@@ -425,13 +431,13 @@ Agrega las siguientes llamadas RPC al espacio de nombres `personal_*`. El valor
...
 
-El gas máximo que consumirá una llamada RPC (utilizado en `eth_estimateGas` y
+El gas máximo que consumirá una llamada RPC (usado en `eth_estimateGas` y
 `eth_call`). El valor predeterminado es `50,000,000`.
 
 #### `rpc-tx-fee-cap` (int)
 
 Límite global de tarifa de transacción (precio \* `gaslimit`) (medido en AVAX) para
-variantes de send-transaction. El valor predeterminado es `100`.
+variantes de envío de transacciones. El valor predeterminado es `100`.
```
Comment (meaghanfitzgerald) [inline i18n/es/docusaurus-plugin-content-docs/current/nodes/configure/chain-config-flags.md:440]:
```suggestion
variantes de send-transaction. El valor predeterminado es `100`.
```

File: i18n/es/docusaurus-plugin-content-docs/current/nodes/configure/chain-config-flags.md [hunk 6]
```diff
@@ -443,7 +449,7 @@ Número máximo de bloques para servir por solicitud de `getLogs`. El valor pred
 
 #### `ws-cpu-refill-rate` (duration)
 
-La tasa de relleno especifica la cantidad máxima de tiempo de CPU que se asignará a una sola
+La tasa de recarga especifica la cantidad máxima de tiempo de CPU que se asignará a una sola
```
Comment (meaghanfitzgerald) [inline i18n/es/docusaurus-plugin-content-docs/current/nodes/configure/chain-config-flags.md:452]:
```suggestion
La tasa de relleno especifica la cantidad máxima de tiempo de CPU que se asignará a una sola
```

File: i18n/es/docusaurus-plugin-content-docs/current/nodes/configure/chain-config-flags.md [hunk 7]
```diff
@@ -576,9 +587,9 @@ Especifica una URI externa para un firmante de tipo clef. El valor predeterminad
 
 #### `keystore-insecure-unlock-allowed` (bool)
 
-Si es `true`, permite desbloquear cuentas en un entorno HTTP inseguro. El valor predeterminado es `false`.
+Si es `true`, permite a los usuarios desbloquear cuentas en un entorno HTTP inseguro. Por defecto, es `false`.
```
Comment (meaghanfitzgerald) [inline i18n/es/docusaurus-plugin-content-docs/current/nodes/configure/chain-config-flags.md:590]:
```suggestion
Si es `true`, permite desbloquear cuentas en un entorno HTTP inseguro. El valor predeterminado es `false`.
```

File: i18n/es/docusaurus-plugin-content-docs/current/nodes/configure/chain-config-flags.md [hunk 8]
```diff
@@ -562,7 +573,7 @@ Define el nivel de registro para la cadena. Debe ser uno de `"trace"`, `"debug"`
 
 Si es `true`, cambia los registros al formato JSON. El valor predeterminado es `false`.
 
-### Configuración de Keystore
+### Configuración de la carpeta de claves
```
Comment (meaghanfitzgerald) [inline i18n/es/docusaurus-plugin-content-docs/current/nodes/configure/chain-config-flags.md:576]:
```suggestion
### Configuración de Keystore
```

File: i18n/es/docusaurus-plugin-content-docs/current/nodes/configure/chain-config-flags.md [hunk 9]
```diff
@@ -610,65 +625,69 @@ Especifica el número máximo de bloques en cola durante la aceptación de bloqu
...
 
-Para anular esto y cambiar al modo de poda, además de `pruning-enabled: true`, `allow-missing-tries` también debe establecerse en `true`.
+Para anular esto y cambiar al modo de recorte, además de `pruning-enabled: true`, `allow-missing-tries` también debe establecerse en `true`.
 
 :::
 
 #### `populate-missing-tries` (\*uint64)
 
-Si no es nulo, establece el punto de partida para repoblar los tries faltantes y regenerar el bosque de merkle de archivo.
+Si no es nulo, establece el punto de partida para repoblar los tries faltantes para regenerar el bosque de merkle de archivo.
```
Comment (meaghanfitzgerald) [inline i18n/es/docusaurus-plugin-content-docs/current/nodes/configure/chain-config-flags.md:644]:
```suggestion
Si no es nulo, establece el punto de partida para repoblar los tries faltantes y regenerar el bosque de merkle de archivo.
```

File: i18n/es/docusaurus-plugin-content-docs/current/nodes/configure/chain-config-flags.md [hunk 10]
```diff
@@ -610,65 +625,69 @@ Especifica el número máximo de bloques en cola durante la aceptación de bloqu
...
+Para restaurar un bosque de merkle de archivo que ha sido corrompido (nodos de trie faltantes para una sección de la cadena de bloques), especifique el punto de partida del último bloque en disco, donde el trie completo estaba disponible en ese bloque para volver a procesar bloques desde esa altura en adelante y regenerar el bosque de merkle de archivo al iniciar. Esta bandera debe usarse una vez para regenerar el bosque de merkle de archivo y debe eliminarse de la configuración después de completarla. Esta bandera hará que el nodo retrase su inicio mientras vuelve a procesar bloques antiguos.
```
Comment (meaghanfitzgerald) [inline i18n/es/docusaurus-plugin-content-docs/current/nodes/configure/chain-config-flags.md:646]:
```suggestion
Para restaurar un bosque de merkle de archivo que ha sido corrompido (nodos de trie faltantes para una sección de la cadena de bloques), especifique el punto de partida del último bloque en disco, donde el trie completo estaba disponible en ese bloque para volver a procesar bloques desde esa altura en adelante y regenerar el bosque de merkle de archivo al iniciar. Esta bandera debe usarse una vez para regenerar el bosque de merkle de archivo y debe eliminarse de la configuración después de completar. Esta bandera hará que el nodo retrase su inicio mientras vuelve a procesar bloques antiguos.
```

File: i18n/es/docusaurus-plugin-content-docs/current/nodes/configure/chain-config-flags.md [hunk 11]
```diff
@@ -610,65 +625,69 @@ Especifica el número máximo de bloques en cola durante la aceptación de bloqu
...
 
 #### `populate-missing-tries-parallelism` (int)
 
 Número de lectores concurrentes a utilizar al volver a poblar los tries faltantes al iniciar. Por defecto, es `1024`.
 
 #### `allow-missing-tries` (boolean)
 
-Si es `true`, permite que un nodo que alguna vez se configuró como de archivo cambie al modo de poda. Por defecto, es `false`.
+Si es `true`, permite que un nodo que alguna vez estuvo configurado como archivador cambie al modo de recorte. Por defecto, es `false`.
```
Comment (meaghanfitzgerald) [inline i18n/es/docusaurus-plugin-content-docs/current/nodes/configure/chain-config-flags.md:654]:
```suggestion
Si es `true`, permite que un nodo que alguna vez se configuró como de archivo cambie al modo de poda. Por defecto, es `false`.
```

**Review by meaghanfitzgerald — DISMISSED** [review]

**Review by meaghanfitzgerald — APPROVED** [review]

**Pull request merged**